server/src/main/java/org/opensearch/index/translog/transfer/TranslogTransferManager.java

```diff
@@ -68,7 +69,7 @@ public TranslogTransferManager(
     ) {
         this.shardId = shardId;
         this.transferService = transferService;
-        this.remoteBaseTransferPath = remoteBaseTransferPath;
+        this.remoteBaseTransferPath = remoteBaseTransferPath.add(DATA_DIR);
```

Comment:
this no longer remains base , but becomes `remoteDataTransferPath` rather.

Reply:
Sure, i'll update the name